openshift/4.0/templates/network-security-policy/internal-k8s-api.yaml

```diff
@@ -1,25 +0,0 @@
-kind: Template
```

Comment:
Similarly, not everyone may agree with this call, but it seemed logical to delete the aporeto nsps if they are being deprecated.